.github/OWNERS

```diff
@@ -39,5 +39,6 @@
 /src/Form/ControlLabel @brandonlilly
 /src/Table @brandonlilly
 /src/Tabs @brandonlilly
+/src/theme/themeMerge @zero0Halo
```

Comment:
Not opposed. I put in with `RootTheme` and `ThemeProvider` because they are all related, but as it's not a component I think it's fine. I can probably get around the circular logic I was encountering by moving it out of the same export, too.